src/qt/qvalidatedlineedit.h

```diff
@@ -29,6 +29,7 @@ class QValidatedLineEdit : public QLineEdit
     const QValidator *checkValidator;
 
 public Q_SLOTS:
+    void setText(const QString&);
```

Comment:
aeb18b665c616c3326671b4c7e9d6421306564f0

Maybe drop this and add in the constructor:
```cpp
connect(this, &QValidatedLineEdit::textChanged, this, &QValidatedLineEdit::checkValidity);
```

Reply:
Then it will run `checkValidity` twice for every keystroke?